### PR TITLE
Add pre-puller.

### PR DESCRIPTION
**Pull request by ryanlovett**
First pass at #33.

### CODE REVIEW

File: pre-pull.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+repo_dirname = os.path.basename(args.repo).split('.')[0]
+
+if not os.path.exists(local_repo):
+    os.mkdir(local_repo)
+
+git_clone()
+
+f = open(args.filename)
+line = f.readline()
+while line != '':
```
Comment (yuvipanda) [inline pre-pull.py]: awesome. did you already run this? :)

Comment (ryanlovett) [inline pre-pull.py]: Yep, last night. I tested it with myself but as you noted it'd be good to test with a username that requires massaging.

**Comment by yuvipanda** [comment]
Let's not call this 'pre-puller' since there's already an image pre-puller. Can you make a dir called scripts and call this 'git-pre-cloner' or something?

**Comment by ryanlovett** [comment]
Replaced by #46.

**Pull request closed without merging**